var/spack/repos/builtin/packages/git/package.py

```diff
@@ -199,6 +200,9 @@ def setup_environment(self, spack_env, run_env):
             spack_env.append_flags('CFLAGS', '-I{0}'.format(
                 self.spec['gettext'].prefix.include))
 
+        # gitk requires tk's wish to be found in $PATH.
+        run_env.prepend_path('PATH', self.spec['tk'].prefix.bin)
```

Comment:
this is only executed at build time, just make the dependency of type `('build', 'run')` and you will be fine.